### PR TITLE
Fix typo in macro use.

### PR DESCRIPTION
**Pull request by mclearn**
Fixes: #19996

CLA: trivial

##### Checklist

- [ ] documentation is added or updated
- [ ] tests are added or updated

### CODE REVIEW

File: crypto/info.c [hunk 1]
```diff
@@ -131,7 +131,7 @@ DEFINE_RUN_ONCE_STATIC(init_info_strings)
 #ifdef OPENSSL_RAND_SEED_NONE
         add_seeds_string("none");
 #endif
-#ifdef OPENSSL_RAND_SEED_RTDSC
+#ifdef OPENSSL_RAND_SEED_RDTSC
         add_seeds_string("stdsc");
```
Comment (t8m) [inline crypto/info.c:135]: Actually the string also should be changed to `"rdtsc"`, this would be still within CLA: trivial.

Comment (mclearn) [reply crypto/info.c]: Thanks. Have made this change. (Of course, the rdtsc source still is prevented from actual use via various preventative mechanisms, so this PR merely fixes up the potential to display the compiled seed source.)

**Review by t8m — APPROVED** [review]

**Comment by t8m** [comment]
OK with CLA: trivial

**Review by hlandau — APPROVED** [review]

**Review by paulidale — APPROVED** [review]

**Comment by t8m** [comment]
Squashed and merged to master, 3.1, and 3.0 branches. Thank you.

**Pull request closed without merging**